content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -23,44 +24,56 @@ Configure payment methods
 -------------------------
 
 To add a payment method, you first need to create it. Go to :menuselection:`Point of Sale -->
-Configuration --> Payment Methods --> Create`. Set a name and select the **payment terminal** or
-check :guilabel:`Cash` for cash payments.
+Configuration --> Payment Methods --> Create`, and set a name. Check :guilabel:`Identify Customer`
+to allow this payment method *exclusively* for registered customers. For any customers to be able to
+pay, leave the :guilabel:`Identify Customer` field unchecked.
+
+Then, select the :guilabel:`Journal`. Choose :guilabel:`Cash` to use this payment method for cash
+payments, or :guilabel:`Bank` to use it for card payments.
+
+.. comment:
+   when created, add a link to the customeraccount page under advanced_pricing_features
 
 .. image:: getting_started/payment-method.png
    :align: center
    :alt: Creating a new payment method for a POS.
 
 .. note::
-   Credentials are mandatory to use a payment terminal. To learn how to configure the different
-   terminals, check out the following documentation pages:.
-
-   - :doc:`Adyen configuration <../payment/adyen>`
-   - :doc:`Vantiv configuration <../payment/vantiv>`
-   - :doc:`Ingenico configuration <../payment/ingenico>`
-   - :doc:`Six configuration <../payment/six>`
-
-Now, you can select the payment method in your POS settings. To do so, go to
+   - Selecting a :guilabel:`bank` journal automatically adds the :guilabel:`Use a Payment Terminal`
+     field in which you can add your terminal's information.
+   - Credentials are mandatory to use a payment terminal. To learn how to configure the different
+     terminals, check out the following documentation pages:
+
+     - :doc:`Adyen configuration <../payment/adyen>`
+     - :doc:`Vantiv configuration <../payment/vantiv>`
+     - :doc:`Ingenico configuration <../payment/ingenico>`
+     - :doc:`Six configuration <../payment/six>`
+     - :doc:`Worldline configuration <../payment/worldline>`
+
+Once the payment method is created, you can select it in your POS settings. To do so, go to
 :menuselection:`Point of Sale --> Configuration --> Point of Sale` and select a POS for which you
 wish to make the payment method available. Click :guilabel:`Edit` and add the payment method under
 the :guilabel:`Payments` section.
 
-
 POS sessions
 ============
 
+.. _session:
```

Comment:
I suggest something more specific.
It's not a rule at all, but that's why I tend to write a prefix for the doc and then the topic.
```suggestion
.. _pos/sessions:
```

also, if it's about the heading "POS sessions", put it _before_ that heading.
If it is for "start a session" I would then suggest something like
```suggestion
.. _pos/start-session:
```